decisions/wc_class_decisions.txt

```diff
@@ -293,7 +293,7 @@ decisions = {
 			is_in_society = no
 			religion = kaldorei_religion
 			is_female = yes
-			reproduces_asexually = no
+			plant_race = no
```

Comment:
Ancients can't be moon priests, but can be warlocks, warriors, shamans. This trigger should be removed from here and added to `is_alternative_class_trigger`

Reply:
Fixed